wagtail/contrib/settings/templates/wagtailsettings/edit.html

```diff
@@ -41,12 +41,14 @@ <h1 class="icon icon-cogs">
 
 {% endblock %}
 
-{% block extra_css %}
+{% block css %}
```

Comment:
Please double check this, I think it should be left as `{% block extra_css %}` - otherwise the documentation will now be wrong.